Aptoide/aptoide-client-v8
# [Feature/ASV-399] Filter appc ads by previous installed

**What does this PR do?**

   AppCoins apps with proof of attention integration are now filtered out if installed or previously installed in the device. Knowing that previous information will be lost if user uninstalls or clear's Aptoide data.

**Database changed?**

   Yes

**Where should the reviewer start?**

- [ ] RemoteBundleDataSource.java

**How should this be manually tested?**

  Install an Appc ad > refresh home > see if it is not present anymore.
  Uninstall the Appc ad > refresh home > see if it is still not present in the bundle.
  Clear data > go to home > see if the app is showing up in the appc bundle.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-399](https://aptoide.atlassian.net/browse/ASV-399)

**Code Review Checklist**
